test/compilable/extra-files/ddoc9764.html

```diff
@@ -464,8 +464,8 @@
     <div class="content_wrapper">
       <article class="module">
         <h1 class="module_name">ddoc9764</h1>
-        <section id="module_content">
-
+        <section id="module_content"><br><br>
+<br><br>

```

Comment:
Hmm, maybe there should be a `strip` or otherwise something to exclude the blank line after the `Ddoc` header